src/js/components/FormField/FormField.js

```diff
@@ -271,6 +272,12 @@ const FormField = forwardRef(
           'FileInput'.indexOf(child.type.displayName) !== -1
         )
           isFileInputComponent = true;
+        if (
+          child &&
+          child.type &&
+          'DateInput'.indexOf(child.type.displayName) !== -1
```

Comment:
Is there a way we can manage this more generically such that FormField doesn't have to specifically know about DateInput and FileInput? This is a type of coupling that causes maintenance issues. It also means if someone creates their own component with similar issues it will be a problem if in a FormField.

Reply:
I'm currently exploring what other ways we can handle this so that we don't have to have specific code to handle the DateInput case

Reply:
I opened a new PR with an alternate solution that is a bit more generic and doesn't rely on specific code to handle the DateInput case https://github.com/grommet/grommet/pull/5895